ark/utils/deepcell_service_utils_test.py

```diff
@@ -58,6 +64,19 @@ def test_create_deepcell_output(mocker):
         os.remove(os.path.join(output_dir, 'fov2_feature_0.tif'))
         os.remove(os.path.join(output_dir, 'example_output.zip'))
 
+        # test parallel
+        create_deepcell_output(deepcell_input_dir=input_dir, deepcell_output_dir=output_dir,
+                               fovs=['fov1', 'fov2'], zip_size=1, parallel=True)
+
+        # make sure DeepCell (.zip's) output exists
+        assert os.path.exists(os.path.join(output_dir, 'example_output.zip'))
+        assert os.path.exists(os.path.join(output_dir, 'example_output_2.zip'))
+
+        os.remove(os.path.join(output_dir, 'fov1_feature_0.tif'))
+        os.remove(os.path.join(output_dir, 'fov2_feature_0.tif'))
```

Comment:
This would be cleaner if we made a new temp folder, rather than deleting the files we've already created. Can either make output1, output2, output3 for each part of the test, or a context manager with a temp dir